packages/atomic/src/components/search/atomic-external/atomic-external.new.stories.tsx

```diff
@@ -0,0 +1,100 @@
+import {Meta, StoryObj} from '@storybook/web-components';
+import {html} from 'lit-html';
+
+const meta: Meta = {
+  component: 'atomic-external',
+  title: 'Atomic/Atomic External',
+  id: 'atomic-external',
+  render: () => html`
+    <style>
+      .wrapper {
+        display: flex;
+      }
+
+      .wrapper > div {
+        padding: 3rem;
+      }
+
+      .wrapper > div * {
+        margin-bottom: 2rem;
+      }
+
+      h1 {
+        font-size: 1rem;
+      }
+    </style>
+    <div class="wrapper">
+      <div>
+        <h1>External components of interface #2</h1>
+        <atomic-external selector="#interface-2">
+          <atomic-search-box></atomic-search-box>
+          <atomic-query-summary></atomic-query-summary>
+          <atomic-facet field="author" label="Author"></atomic-facet>
+        </atomic-external>
+      </div>
+      <div>
+        <h1>Interface #1, not linked to URL</h1>
+        <atomic-search-interface
+          id="interface-1"
+          pipeline="UI_KIT_E2E"
+          search-hub="UI_KIT_E2E"
+          reflect-state-in-url="false"
+        >
+          <atomic-query-summary></atomic-query-summary>
+          <atomic-numeric-facet
+            field="ec_price"
+            label="Cost"
+            with-input="integer"
+          >
+            <atomic-format-currency currency="USD"></atomic-format-currency>
+          </atomic-numeric-facet>
+          <atomic-search-box></atomic-search-box>
+          <atomic-result-list></atomic-result-list>
+        </atomic-search-interface>
+      </div>
+      <div>
+        <h1>Interface #2, linked to URL</h1>
+        <atomic-search-interface id="interface-2">
+          <atomic-query-summary></atomic-query-summary>
+          <atomic-result-list></atomic-result-list>
+        </atomic-search-interface>
+      </div>
+    </div>
+  `,
+  play: async () => {
+    await customElements.whenDefined('atomic-search-interface');
+
+    const searchInterface1 = document.querySelector(
+      '#interface-1'
+    ) as HTMLAtomicSearchInterfaceElement;
+    const searchInterface2 = document.querySelector(
+      '#interface-2'
+    ) as HTMLAtomicSearchInterfaceElement;
+
+    await Promise.all([
+      searchInterface1.initialize({
+        accessToken: 'xxc23ce82a-3733-496e-b37e-9736168c4fd9',
+        organizationId: 'electronicscoveodemocomo0n2fu8v',
```

Comment:
Again, minor bother: I'm not a fan of show-casing a commerce use-case that doesn't rely on the atomic-commerce components.

One thing you could do would be to point to `searchuisamples`, but use an accessToken (i.e., API key) that enforces a different search hub. You'd route queries with that search hub to a dedicated pipeline which could enforce a specific filter expression (e.g., to target content from a specific source).

This also doesn't have to be done right away, but I think it would be better in the end if we had no commerce samples in storybook outside of the atomic-commerce section.

Reply:
https://coveord.atlassian.net/browse/KIT-3607